specification/iotoperations/IoTOperations.Management/tspconfig.yaml

```diff
@@ -0,0 +1,14 @@
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    emitter-output-dir: "{project-root}/.."
+    azure-resource-provider-folder: "resource-manager"
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/iotoperations.json"
+    examples-directory: "{project-root}/examples"
+    use-read-only-status-schema: true
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/resource-manager"
+  disable:
+    "@azure-tools/typespec-azure-rulesets/resource-manager/arm-common-types-version": "New rule"
```

Comment:
>@azure-tools/typespec-azure-rulesets/resource-manager/arm-common-types-version": "New rule" [](http://example.com/codeflow?start=5&length=91)

is this necessary?

Reply:
i don't recall why we included this, removed it for now to see if typespec tests pass